### PR TITLE
refactor the report to render amount with enterprise currency

### PR DESCRIPTION
**Pull request by DedrickEnc**
This P.R fix an issue relative to the currency, now the income expense can render amount in the enterprise currency and not a default one.
inc-exp-currency

### CODE REVIEW

**Comment by DedrickEnc** [comment]
@sfount 
The changes are landed

**Review by sfount — APPROVED** [review]
Code LGTM 👍 Great quick fix.

**Comment by DedrickEnc** [comment]
@sfount
can you merge it?

File: server/controllers/finance/reports/incomeExpense/report.handlebars [hunk 1]
```diff
@@ -38,16 +38,16 @@
                 <tr>
                   <td>{{number}}</td>
                   <td>{{label}}</td>
-                  <td class="text-right">{{currency debit}}</td>
-                  <td class="text-right">{{currency credit}}</td>
+                  <td class="text-right">{{currency debit ../enterprise.currency_id}}</td>
```
Comment (jniles) [inline server/controllers/finance/reports/incomeExpense/report.handlebars:41]: Just FYI, I believe the enterprise is always available on the `metadata` object.  For example, 
```html
<td class="text-right">{{currency debit metadata.enterprise.currency_id}}</td>
```

You can see this usage in [the invoices report](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/controllers/finance/reports/invoices/report.handlebars#L45).

**Review by jniles — APPROVED** [review]

**Pull request merged**